docs/tasks/argo.rst

```diff
@@ -0,0 +1,253 @@
+``````````````````````````````````````````````````
+
+Here, you will reference the MySQL_ example from Kanister.
+Install the Chart and setup MySQL in the ``mysql-test`` namespace.
+Make sure that the application is integrated with Kanister and the following
+steps are completed.
+
+1. A Profile CR is created for the ``mysql-test`` namespace.
+2. A MySQL blueprint is created in the ``kanister`` namespace.
+3. Check the name of the blueprint, stateful set, profile and
+   secrets for you application for the next step.
+
+Step 3 - Creating a Cron Workflow
+````````````````````````````````````````````
+
+Now a Cron Workflow will be created that will automate the creation of an actionset
+to backup our MySQL application. You will need to modify the names of the blueprint,
+stateful set, profile and secrets as per the ones you have created.
+Modify the last line of the following yaml file as follows.
+
+.. code-block:: bash
+
+  kanctl create actionset --action backup --namespace kanister --blueprint [YOUR_BLUEPRINT_NAME] --statefulset [YOUR_STATEFULSET] --profile [YOUR_PROFILE_NAME] --secrets [YOUR_SECRETS_NAME]
+
+Then execute -
+
+.. code-block:: bash
+
+  cat<<EOF >> mysql-cron-wf.yaml
+  apiVersion: argoproj.io/v1alpha1
+  kind: CronWorkflow
+  metadata:
+    name: mysql-cron-wf
+  spec:
+    schedule: "*/5 * * * *"
+    concurrencyPolicy: "Replace"
+    workflowSpec:
+      entrypoint: automate-actionset
+      templates:
+        - name: automate-actionset
+          container:
+            image: ghcr.io/kanisterio/kanister-tools:v9.99.9-dev
+            command:
+              - /bin/bash
+              - -c
+              - |
+                microdnf install tar
+                curl -LO https://github.com/kanisterio/kanister/releases/download/0.80.0/kanister_0.80.0_linux_amd64.tar.gz
+                tar -C /usr/local/bin -xvf kanister_0.80.0_linux_amd64.tar.gz
+                kanctl create actionset --action backup --namespace kanister --blueprint mysql-blueprint --statefulset mysql-test/mysql-release --profile mysql-test/s3-profile-gd4kx --secrets mysql=mysql-test/mysql-release
+  EOF
+
+Note - Here, the cron-job is scheduled to run every 5 minutes. You may schedule
+it to run as per your requirements.
+
+Step 4 - Granting RBAC permissions.
+````````````````````````````````````````````
+
+Next, you will grant the Service Account in our ``argo`` namespace to access resources
+in the ``kanister`` and ``mysql-test`` namespace. This is required to create CRs based on
+the Secrets and StatefulSets that you provided in our previous step.
+You may read more about RBAC authorization here - RBAC_.
+
+1. Create a ClusterRole named ``cron-wf-manager`` having the permissions in
+   ``cluster-admin``.
+2. Grant these permissions to the default ServiceAccount named
+   ``default`` in the ``argo`` namespace.
+3. This will be done by creating a RoleBinding in the ``kanister``
+   and ``mysql-test`` namespaces.
+
+Execute the following -
+
```

Comment:
did we miss the step to create the `clusterrole`?

Reply:
No. The bullet points above explain what is happening in the single command.

Reply:
for these three points can we mention the command inline. Means, a bullet point and then command.

Reply:
You mean parts of the command? They have been highlighted in the bullet points itself so it would be repetitive.  And for further reference, I have provided the link to the K8s RBAC documentation.

Reply:
Done.